crates/re_types/src/archetypes/points3d_ext.rs

```diff
@@ -0,0 +1,214 @@
+use super::Points3D;
+
+impl Points3D {
+    /// Creates a new [`Points3D`] from a `.ply` file.
+    ///
+    /// ## Supported properties
+    ///
+    /// This expects the following property names:
+    /// - (Required) Positions of the points: `"x"`, `"y"` & `"z"`.
+    /// - (Optional) Colors of the points: `"red"`, `"green"` & `"blue"`.
+    /// - (Optional) Radii of the points: `"radius"`.
+    /// - (Optional) Labels of the points: `"label"`.
+    ///
+    /// The media type will be inferred from the path (extension), or the contents if that fails.
+    #[cfg(not(target_arch = "wasm32"))]
+    pub fn from_file(filepath: &std::path::Path) -> anyhow::Result<Self> {
```

Comment:
```suggestion
    pub fn from_file_pathfilepath: &std::path::Path) -> anyhow::Result<Self> {
```

Reply:
why I insist on this btw: I've had so many egui users misunderstand data loading so many times. I have a function for loading a texture from a width, height, and `&[u8]` of RGBA pixels, and I sweat to you multiple people try to put the encoded contents  of a `.png` in it as well as a file path in UTF8.

point is: better to be over explicit in these things, even when there is type safety